dumper/dataWriter.go

```diff
@@ -445,6 +445,19 @@ func formatOnConflict(row []sqlUtil.RowDataStructure, table schemareader.Table)
 		} else {
 			constraint = "(advisory, org_id) WHERE org_id IS NOT NULL"
 		}
+	case "rhnerratacve":
+		var errataId interface{} = nil
+		for _, field := range row {
+			if strings.Compare(field.ColumnName, "errata_id") == 0 {
+				errataId = field.Value
+				break
+			}
+		}
+		if errataId == nil {
+			constraint = "(cve_id) WHERE errata_id IS NULL"
```

Comment:
I think this will not work since the table doesn't have a unique constraint with this format. It only has a unique constraint on `(errata_id, cve_id)`

Reply:
Thanks for checking it. So do you think if I change `rhn_err_cve_cid_idx` to unique index might work?

Reply:
Can that be unique? Can we say that a CVE ID is only associated to one errata ID? How about CVE's that affect several different packages?
That will not work :) 
We need to think about a different solution.